docs/external plugins/Verifier/sbom.md

```diff
@@ -0,0 +1,294 @@
+          "extensions": {
+            "Issuer": "CN=wabbit-networks.io,O=Notary,L=Seattle,ST=WA,C=US",
+            "SN": "CN=wabbit-networks.io,O=Notary,L=Seattle,ST=WA,C=US"
+          },
+          "artifactType": "application/vnd.cncf.notary.signature"
+        }
+      ],
+      "artifactType": "application/spdx+json"
+    }
+  ]
+}
+```
+## Configuration
+Sample YAML
+```json
+apiVersion: config.ratify.deislabs.io/v1beta1
+kind: Verifier
+metadata:
+  name: verifier-sbom
+spec:
+  name: sbom
+  artifactTypes: application/spdx+json
+  parameters:
+    disallowedPackages:
+    - name: busybox
+      version: 1.36.1-r0
+    disallowedLicenses: 
+    - MPL
+```
+| Name                  | Required | Path                                  | Description                                                                                                                                                                                          | Default Value                      |
+| --------------------- | -------- | ------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------- |
+| disallowedPackages            | No       | spec.parameters.disallowedPackages            | Array of disallowed packages. If version is empty, all packages with matching name will be disallowed.                                                                                                                                                              | []                                |
+| disallowedLicenses  | No       | spec.parameters.disallowedLicenses  | String array of disallowed licenses.  | []                                 |
+
+### CLI
+
+Sample JSON
+```json
+{
+    "store": {
+        "version": "1.0.0",
+        "plugins": [
+            {
+                "name": "oras",
+                "useHttp": true
+            }
+        ]
+    },
+    "policy": {
+        "version": "1.0.0",
+        "plugin": {
+            "name": "configPolicy",
+            "artifactVerificationPolicies": {
+                "application/spdx+json": "all"
+            }
+        }
+    },
+    "verifier": {
+        "version": "1.0.0",
+        "plugins": [
+            {
+                "name": "sbom",
+                "artifactTypes": "application/spdx+json",
+                "disallowedLicenses": ["MPL"],
+                "disallowedPackages":[{"name":"busybox","version":"1.36.1-r0"}]
+            }
+        ]
+    }
+}
+```
+### Future Improvements
+Please vote on these issues to help us prioritize:
+- [SBOM Verifier to support other formats](https://github.com/deislabs/ratify/issues/1206)
+- [SBOM verifier to support license expressions](https://github.com/deislabs/ratify/issues/1207)
+
+### FAQ
+
+#### Why are there multiple external verifiers that can verify SBOMs?
```

Comment:
Maybe add a note about future deprecation of license checker. recommended guidance is to use SBOM verifier going forward

Reply:
The `licensechecker` verifier has been DEPRECATED and will be removed in future releases. Please use the SBOM verifier for license checks moving forward. Package license verification is associated typically with SBOMs. As such, Ratify has decided to incorporate package license filtering in SBOM verification.